File: backend/app/db/models/exercise.py
"""Таблица «Упражнение» (Exercise).

Справочник доступных упражнений (например, «Румынская тяга»). На
текущем шаге — пустой класс. Поля (имя, муск.группа, ссылка на видео-
демо и т.д.) появятся в Phase 2 / thin slice.
"""

from sqlmodel import Field, SQLModel


class Exercise(SQLModel, table=True):
    """Справочник упражнений. Поля будут добавлены в thin slice."""

    id: int | None = Field(default=None, primary_key=True)

File: web/src/domain/exercise.ts
/**
 * Доменный тип Exercise (упражнение).
 *
 * Это TS-зеркало серверной модели `backend/app/db/models/exercise.py`.
 * Сейчас обе стороны пустые: серверная — `class Exercise(SQLModel, table=True): pass`,
 * клиентская — пустой объект. Поля появятся в Phase 2 (thin slice).
 *
 * Пример (НЕ актуально сейчас, для иллюстрации формы):
 *   {
 *     id: 'squat',
 *     name: 'Приседания',
 *     videoUrl: 'https://…',
 *   }
 */
export type Exercise = Record<string, never>;
